src/container/container_iv.c

```diff
@@ -653,8 +683,18 @@ cont_iv_ent_update(struct ds_iv_entry *entry, struct ds_iv_key *key,
 		} else if (entry->iv_class->iv_class_id ==
 						IV_CONT_AGG_EPOCH_BOUNDRY) {
 			rc = cont_iv_ent_agg_eph_refresh(entry, key, src);
-			if (rc)
+			if (rc) {
+				/* container non-exist possibly due to in reintegrate the container
+				 * discarded ahead. Ignore such err.
+				 */
+				if (rc == -DER_NONEXIST) {
```

Comment:
You have already changed `cont_discard_cb()` to prevent such case, so the container will not be discarded for reintegration case?

Reply:
ah right, such code is harmless although, for some orphan container cases may be useful. The comment should change, will change it later if need to refresh for some other issue. Mohamad's test still with a few problems.